src/hammr/utils/credentials_utils.py

```diff
@@ -0,0 +1,154 @@
+__author__ = 'UshareSoft'
+import os
+import json
+import getpass
+
+from hammr.utils import printer, generics_utils
+
+class Credential:
+    '''
+    This class get user credentials fron command line or config file
+    It also check if the mandatory info are present
+    '''
+    def __init__(self):
+        self.sslAutosigned = True
+        self.password = None
+        self.userpassAuthentication = False
```

Comment:
I understood that I will have to create a function that will use these two booleans (still keep them), is it right ?